src/pet_to_stp.jl

```diff
@@ -46,6 +46,9 @@ function pet_to_stp(pet_table::Table, detector_SSD::SolidStateDetector)
 
     hits = RadiationDetectorSignals.ungroup_by_evtno(events_clustered)
 
+    @info "...removing hits with no energy deposits"
+    hits = hits[findall(edep -> !iszero(edep), hits.edep)]
```

Comment:
Not sure whether it's more efficient to use `filter!` here: `filter!(hits, h -> !iszero(h.edep))`
@oschulz ?